docs/contributing/index.md

```diff
@@ -55,14 +55,59 @@ If you don't get a timely reply, ask the documentation owners for a review or fo
 
 ## Guidelines
 
-### Frequently Asked Questions
+### Links
 
-The documentation does not have a FAQ section, because questions are best answered by the documentation, not in a separate section.
-Integrating information into the main documentation requires some care to identify where the information needs to go, and edit the documentation around it.
-Adding the information to a FAQ is easier, but the result is information about a topic distributed betwen the docs and FAQ questions, which ultimately makes the documentation harder to search.
+#### external links
 
-FAQ content, such as lists of most frequently encountered error messages, is still very useful in many contexts.
-If you want to add such content, create a section at the bottom of a topic page, for example this section on the [SSH documentation page][ref-ssh-faq].
+Links to external sites use the `[]()` syntax:
+
+=== "external link syntax"
+
+    ```
+    [The Spack repository](https://github.com/spack/spack)
+    ```
+
+=== "result"
+
+    [The Spack repository](https://github.com/spack/spack)
+
+#### internal links
+
+!!! note
+    The CI/CD pipeline will fail if it detects broken links in your draft documentation.
+    it is not completely foolproof - to ensure that your changes do not create broken links you should merge the most recent version of the `main` branch of the docs into your branch branch.
```

Comment:
```suggestion
    It is not completely foolproof - to ensure that your changes do not create broken links you should merge the most recent version of the `main` branch of the docs into your branch branch.
```

Reply:
Note to self: use merge queue to avoid this situation.

https://github.com/eth-cscs/cscs-docs/issues/55